### DISCUSSION TITLE
Which source file / function does "electrolyte conductivity" use for "default" option

### DISCUSSION THREAD

**Discussion by RuiheLi** [pybamm-team/PyBaMM#2650] [Q&A]
Dear PyBaMM team,

In pybamm\models\full_battery_models\base_battery_model.py, the default option for the "electrolyte conductivity" is default. But in pybamm\models\submodels\electrolyte_conductivity, I cannot find any source files that seems to relate to "default". For DFN, it seems to use "Full" conductivity. May I ask which source file / function does "electrolyte conductivity" use for "default" option?

Yours Sincerely,
Ruihe

**Comment by brosaplanella** [comment]
Hi Ruihe! It depends on whether you choose DFN or SPMe. For DFN it uses the Full model (see line 108 of [DFN model](https://github.com/pybamm-team/PyBaMM/blob/develop/pybamm/models/full_battery_models/lithium_ion/dfn.py)) and for SPMe it uses the Composite model (see line 68 of [SPMe](https://github.com/pybamm-team/PyBaMM/blob/develop/pybamm/models/full_battery_models/lithium_ion/spme.py)).
